sys/net/gnrc/network_layer/ipv6/nib/_nib-internal.h

```diff
@@ -98,6 +102,8 @@ typedef struct _nib_onl_entry {
      * @note    Only available if @ref GNRC_IPV6_NIB_CONF_ARSM != 0.
      */
     uint8_t l2addr[GNRC_IPV6_NIB_L2ADDR_MAX_LEN];
+    evtimer_msg_event_t reach_timeout;  /**< Event for @ref GNRC_IPV6_NIB_REACH_TIMEOUT */
+    evtimer_msg_event_t delay_timeout;  /**< Event for @ref GNRC_IPV6_NIB_DELAY_TIMEOUT */
```

Comment:
We could scrape of 4 bytes per event, if we use e.g. cib for the event queue of `evtimer`, but that's an optimization of `evtimer` (or its later replacement), not NIB.